pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -160,9 +160,9 @@ public BaseBrokerRequestHandler(PinotConfiguration config, RoutingManager routin
     _numDroppedLogRateLimiter = RateLimiter.create(1.0);
 
     _brokerReduceService = new BrokerReduceService(_config);
-    LOGGER.info(
-        "Broker Id: {}, timeout: {}ms, query response limit: {}, query log length: {}, query log max rate: {}qps",
-        _brokerId, _brokerTimeoutMs, _queryResponseLimit, _queryLogLength, _queryLogRateLimiter.getRate());
+    LOGGER
```

Comment:
Can you please check if you import the latest `Pinot Style`? There are lots of reformatting changes and seems they don't follow the `Pinot Style`: https://docs.pinot.apache.org/developers/developers-and-contributors/code-setup#setup-ide

Reply:
@Jackie-Jiang  I just double checked: 
* delete the `Pinot Style` from my `Intellj->Editor -> Code Style -> Java`
* import codestyle-intellij.xml from pinot/config again.
* reformat the file using the new `Pinot Style`

Nothing changed. Which means the reformatting in the PR followed `Pinot Style`, while the old file not. 

Could you please do me a favor by importing the latest `Pinot Style` and perform `Reformat code` against this file using your IntellJ (using the current master branch)? 
* If your file does not change, then maybe my IntellJ have some issues, I will revert those reformatting change
* If your file also get changed, let's keep those reformatting change in the PR.

Does it sounds good?

Reply:
I actually tried that before leaving the comment and the current master file didn't change.
In the `Editor -> Code Style -> Java` setting, under `Wrapping and Braces -> Keep when reformatting`, can you uncheck the `Line breaks` and try reformatting again? Current setting preserves the line break when reformatting because in certain cases we want to break the line in a different way.

Reply:
Updated. Thanks!